src/js/containers/projectValidation/ProjectValidationContainer.js

```diff
@@ -24,10 +24,6 @@ import {withLocale} from '../Locale';
 class ProjectValidationContainer extends Component {
   shouldComponentUpdate(nextProps) {
     const {loadingProject} = nextProps.reducers.homeScreenReducer;
-    const { showProjectValidationStepper } = nextProps.reducers.projectValidationReducer;
-    if (showProjectValidationStepper === true) {
-      return true;
-    }
     if (loadingProject === true) {
```

Comment:
Should `loadingProject` be removed as well?

Reply:
It looks like it is getting removed in the other PR.  https://github.com/unfoldingWord-dev/translationCore/pull/6397/files#diff-f413ef91441c3a0ad6e44e8adbdf8389
Whichever you want to merge first I will check for merge conflicts and make sure its all good.

Reply:
Merge w/ latest dev branch

Reply:
Done.

Reply:
@RoyalSix it still looks like is there?